src/teleop/gui/src/components/EndEffectorUV.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="wrap">
+      <div>
+        <h3> End Effector UV </h3>
+      </div>
+      
+    <div>
+      <ToggleButton id="uv_end_effector" v-bind:currentState="UV_endEffector" labelEnableText="End Effector UV On" labelDisableText="End Effector UV Off" v-on:change="toggleEndEffectorUV()" />
+    </div>
+    <br>
+    <div>
+      <ToggleButton id="shutdown" v-bind:currentState="shutdownActive" labelEnableText="UV Auto shutoff On" labelDisableText="UV Auto shutoff Off" v-on:change="switchShutdown()"/>
+    </div>
+  
+    <!-- <div>
+      <ToggleButton id="scoop_limit_switch" v-bind:currentState="scoopLimitActive" labelEnableText="Limit Switch On" labelDisableText="Limit Switch Off" v-on:change="toggleLimit()"/>
+    </div> -->
+  </div>
+</template>
+    
+<script>
+  import ToggleButton from './ToggleButton.vue'
+  import ROSLIB from 'roslib';
+
+  export default {
+    data () {
+      return {
+        UV_endEffector: false,
+        shutdownActive: true,
+        timeoutID: 0,
+        scoopLimitActive: true,
+
+        // Pubs and subs
+        uvService: null
+      }
+    },
+
+    components: {
+      ToggleButton
+    },
+
+    watch: {
+      UV_endEffector() {
```

Comment:
Does eslint not specify a way to define watch functions? I feel like it's clearest as in the first answer here: https://stackoverflow.com/questions/44584292/how-to-listen-for-props-changes

But generally I think we should just be consistent